casper/src/test/scala/coop/rchain/casper/engine/Setup.scala

```diff
@@ -131,7 +136,14 @@ object Setup {
     implicit val synchronyConstraintChecker   = SynchronyConstraintChecker[Task](0d)
     implicit val lastFinalizedConstraintChecker =
       LastFinalizedHeightConstraintChecker[Task](Long.MaxValue)
-    implicit val estimator = Estimator[Task](Estimator.UnlimitedParents, None)
+    implicit val estimator      = Estimator[Task](Estimator.UnlimitedParents, None)
+    implicit val blockRetriever = BlockRetriever.of[Task]
+
+    implicit val kvsManager = RNodeKeyValueStoreManager[Task](Path.of("tmp"))
+      .unsafeRunSync(monix.execution.Scheduler.Implicits.global)
```

Comment:
Path for should be to system temporary directory. With `Path.of("tmp")` this is relative to the current executing directory.
https://github.com/rchain/rchain/blob/0dd10620378443aaa90da7dba3270202e437c7f2/casper/src/test/scala/coop/rchain/casper/engine/Setup.scala#L44

Another option is to use `InMemoryStoreManager[Task]` which is not persisted on disk.